tools/shared.py

```diff
@@ -265,6 +265,12 @@ def generate_config(path, first_time=False):
 # 4. Fall back users home directory (~/.emscripten).
 
 embedded_config = path_from_root('.emscripten')
+# For compatibility with emsdk --embedded mode also look two levels up.  This
```

Comment:
maybe mention why two levels up? That is, that when using the emsdk, the structure would look like
```
emsdk/
  emscripten/
```
But that picture must be wrong since you say two levels and not one..?